lib/va_profile/models/service_history.rb

```diff
@@ -80,6 +80,30 @@ def self.build_from_academy_episode(episode)
           end_date: episode['academy_end_date']
         )
       end
+
+      def self.determine_eligibility(episodes)
+        problem_message = [
+          'We’re sorry. There’s a problem with your discharge status records. We can’t provide a Veteran status ' \
+          'card for you right now.',
+          'To fix the problem with your records, call the Defense Manpower Data Center at 800-538-9552 (TTY: 711).' \
+          ' They’re open Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.'
+        ]
+        not_eligible_message = [
+          'Our records show that you’re not eligible for a Veteran status card. To get a Veteran status card, you ' \
+          'must have received an honorable discharge for at least one period of service.',
+          'If you think your discharge status is incorrect, call the Defense Manpower Data Center at 800-538-9552 ' \
+          '(TTY: 711). They’re open Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.'
+        ]
```

Comment:
Not a blocker but I would recommend moving these messages out of the function and into constants on the service object.  Then they can be referenced in the tests.

Reply:
Totally considered the constants, but decided against it because this is temporary code that will get pulled out hopefully in a couple months when we change the API. Therefore, kind of wanted to keep it contained.